org.eclipse.winery.repository.rest/src/main/java/org/eclipse/winery/repository/rest/resources/refinementmodels/TopologyFragmentRefinementModelResource.java

```diff
@@ -127,6 +129,16 @@ public TopologyTemplateResource savePrmMappingTopology(TTopologyTemplate topolog
                     .setArtifactType(QName.valueOf(propertiesMap.get("requiredDeploymentArtifactType")))
                 ));
             }
+            if (relTemplate.getType().getLocalPart().startsWith("BehaviorPatternMapping")) {
```

Comment:
Maybe also ignore the case...

Reply:
I'm not exactly sure what you mean, if I ignore this case the Behavior Pattern Mappings won't be saved in the Grafic PRM view. Or should I move the case to the `PatternRefinementModelResource`?